extensions/INTEL/SPV_INTEL_fpga_memory_attributes.asciidoc

```diff
@@ -199,4 +211,5 @@ None.
 |E|2019-06-18|Joe Garvey|Added the MaxReplicatesINTEL, SimpleDualPortINTEL, and MergeINTEL decorations
 |F|2019-12-18|Julian Packer|Added the BankBitsINTEL decoration
 |G|2020-02-06|Mohammad Fawaz|Added the ForcePow2DepthINTEL decoration
+|H|2023-07-26|Artem Radzikhovskyy|Added StridesizeINTEL, WordsizeINTEL, TrueDualPortINTEL decorations
```

Comment:
Minor: I think the version section at the top of this document should be updated from G -> H also?